awx/ui_next/src/screens/Template/shared/WebhookSubForm.test.jsx

```diff
@@ -99,7 +100,7 @@ describe('<WebhooksSubForm />', () => {
               webhook_key: 'A NEW WEBHOOK KEY WILL BE GENERATED ON SAVE.',
             }}
           >
-            <WebhookSubForm enableWebhooks />
+            <WebhookSubForm enableWebhooks templateType="job_template" />
           </Formik>
         </Route>,
         {
```

Comment:
might be good to have a test for the workflow templateType too

Reply:
Yeah I agree with @jlmitch5